{{cookiecutter.project_slug}}/_/ci-services/deploy-stage/.gitlab-ci.yml

```diff
@@ -3,31 +3,25 @@ development:
   extends: .deploy
   environment:
     name: development
-    url: https://console.appuio.ch/console/project/{{ cookiecutter.project_slug }}-development/overview
   variables:
     SOURCE: {{ cookiecutter.project_slug }}-development
-    TARGET: {{ cookiecutter.project_slug }}-development
```

Comment:
Both values must depend on the deployment strategy. `SOURCE` cannot stay suffixed. Same below in this file.

Reply:
The `SOURCE` is only rendered for dedicated environments now (only needed for image tagging).